canvas_modules/common-canvas/src/tooltip/tooltip.jsx

```diff
@@ -522,14 +540,17 @@ ToolTip.propTypes = {
 	showToolTipIfTruncated: PropTypes.bool, // Set to true to only display tooltip if full text does not fit in displayable width
 	truncatedRef: PropTypes.object,
 	delay: PropTypes.number,
-	showToolTipOnClick: PropTypes.bool
+	showToolTipOnClick: PropTypes.bool,
+	hoverable: PropTypes.bool, // if true, can hover over to the tooltip, instead of immedieatly disappearing.
+	inTooltip: PropTypes.bool
```

Comment:
You are declaring `inTooltip` here as a prop for the Tooltip object but you are using it as `this.inTooltip`. There's a difference between a prop and member variables of a React object. You should remove this from here and declare `this.inTooltip` in the constructor -- with a comment to explain the purpose of the variable.  (Note: it's not strictly necessary to declare it, since it will default to undefined (falsey) when you first use it, but declaring in in the constructor given you the opportunity to comment on its usage)